packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -212,24 +212,14 @@ export class EVMAAWallet extends LoggerWrapper {
         });
     }
 
-    public getSigner(type: "viem" = "viem"): {
-        publicClient: PublicClient;
-        walletClient: SmartAccountClient<EntryPoint, HttpTransport, Chain, SmartAccount<EntryPoint>>;
+    public get client(): {
```

Comment:
Meant to interop w/ `viem` seamlessly. For example, with this method, interacting with a contract would look like:

```TS
const wallet = await sdk.getOrCreateWallet({userId: "joebob", jwt: "123"}, "base");
const contract = getContract({
  address: '0xFBA3912Ca04dd458c843e2EE08967fC04f3579c2',
  abi: nftCollection,
  client: wallet.client
})
const hash = await contract.mint()
```